packages/idl/package.json

```diff
@@ -12,6 +12,6 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "webidl2": "^23.13.1"
+    "webidl2": "^24.0.1"
```

Comment:
I didn't bother here but this can also be a range that covers both 23 and 24.

Reply:
I think that's the right call. To continue to have confidence in that we'd need to test both versions, and that's not worth it.